tests/integration/piv_test.py

```diff
@@ -252,6 +252,175 @@ def _check_db_stomp_20201022_23186_delayed_5min():
+        assert second_st.departure_delay == timedelta(minutes=5)
+        assert second_st.departure_status == ModificationType.update.name
+        assert second_st.message == "Absence inopinée d'un agent (motifModification depart)"
+
+        third_st = db_trip_delayed.stop_time_updates[2]
+        assert third_st.stop_id == "stop_point:PIV:85010140:Train"
+        assert third_st.arrival == datetime(2020, 10, 22, 20, 47)
+        assert (
+            third_st.arrival_status == ModificationType.update.name
+        )  # as feed status is "RETARD_PROJETE". OK to be corrected to 'none' in the future.
+        assert third_st.arrival_delay == timedelta(minutes=0)
+        assert third_st.departure == datetime(2020, 10, 22, 20, 48)
+        assert third_st.departure_delay == timedelta(minutes=0)
+        assert (
+            third_st.departure_status == ModificationType.update.name
+        )  # as feed status is "RETARD_PROJETE". OK to be corrected to 'none' in the future.
+        assert third_st.message == "Absence inopinée d'un agent (motifModification arrivee)"
+
+        fourth_st = db_trip_delayed.stop_time_updates[3]
+        assert fourth_st.stop_id == "stop_point:PIV:85162735:Train"
+        assert fourth_st.arrival == datetime(2020, 10, 22, 21, 16)
+        assert fourth_st.arrival_status == ModificationType.none.name
+        assert fourth_st.arrival_delay == timedelta(minutes=0)
+        assert fourth_st.departure == datetime(2020, 10, 22, 21, 17)
+        assert fourth_st.departure_delay == timedelta(minutes=0)
+        assert fourth_st.departure_status == ModificationType.none.name
+        assert fourth_st.message == "Absence inopinée d'un agent"
+
+        fifth_st = db_trip_delayed.stop_time_updates[4]
+        assert fifth_st.stop_id == "stop_point:PIV:85162750:Train"
+        assert fifth_st.arrival == datetime(2020, 10, 22, 21, 19)
+        assert fifth_st.arrival_status == ModificationType.none.name
+        assert fifth_st.arrival_delay == timedelta(minutes=0)
+        assert fifth_st.departure == datetime(2020, 10, 22, 21, 20)
+        assert fifth_st.departure_delay == timedelta(minutes=0)
+        assert fifth_st.departure_status == ModificationType.none.name
+        assert fifth_st.message == None
+
+        last_st = db_trip_delayed.stop_time_updates[5]
+        assert last_st.stop_id == "stop_point:PIV:87745497:Train"
+        assert last_st.arrival == datetime(2020, 10, 22, 21, 25)
+        assert last_st.arrival_status == ModificationType.none.name
+        assert last_st.arrival_delay == timedelta(minutes=0)
+        # no specific functional constraint on last departure, except time consistency
+        assert last_st.arrival <= last_st.departure
+        assert last_st.message == None
+
+        assert db_trip_delayed.contributor_id == PIV_CONTRIBUTOR_ID
+
+        return db_trip_delayed  # for additional testing if needed
+
+
+def _check_db_stomp_20201022_23187_delayed_5min():
```

Comment:
Would it be possible to merge this one with the other function that tests `delayed_5min`, I guess it would reduce the number of function, probably reduce the number of stops from the other function (17 to 6). Also, I'm still thinking these functions should be named like this

```suggestion
def _assert_db_stomp_20201022_23187_delayed_5min():
```

Reply:
I renamed it (didn't remember that valid suggestion until now).

It can't be merged as-is though, as the third stop is 10min later: to be able to have the second stop 5min late but not inconsistent in stop-times.
As the stops in the other one are partly tested in a loop and also lots are used to test messages...
I'm not super motivated to merge those :tired_face: